jsapp/js/components/formEditors.es6

```diff
@@ -232,7 +255,7 @@ export class ProjectDownloads extends React.Component {
                           <label htmlFor='lang'>{t('Value and header format')}</label>
                           <select name='lang' value={this.state.lang}
                               onChange={this.langChange}>
-                            <option value='xml'>{t('XML values and headers')}</option>
+                            <option value='_xml'>{t('XML values and headers')}</option>
```

Comment:
Admirable, but this wraps onto a second line:
![image](https://user-images.githubusercontent.com/2085013/47331818-54e53980-d64b-11e8-9cb1-63be43588239.png)

Also, for every language it's the `values and headers`, but we probably wouldn't want to say `English (en) values and headers` either.